src/components/devices/json/counter.vue

```diff
@@ -51,6 +51,79 @@
 				simuliert.
 			</template>
 		</openwb-base-text-input>
+		<openwb-base-text-input
```

Comment:
```
		<openwb-base-alert subtype="info">
			Werden sowohl Leistung als auch Strom auf den Einzelphasen leer gelassen, erfolgt das                   Lastmanagement am EVU-Punkt nur anhand der Gesamtleistung am EVU-Punkt. Wird der Zähler als Zwischenzähler verwendet, wird in diesem Fall kein Lastmanagement durchgeführt.
		</openwb-base-alert>
```
Statt in allen Hilfetexten den gleichen Text zu schreiben, bitte einen Info-Kasten über den betreffenden Einstellungen platzieren (zwischen Z53 und 54).

Reply:
Änderungsvorschläge übernommen